schemas/session-replay/_common-shape-wireframe-schema.json

```diff
@@ -0,0 +1,58 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "_common-shape-wireframe-schema.json",
+  "title": "CommonShapeWireframe",
+  "type": "object",
+  "description": "Schema of common properties for ShapeWireframe events type and all its sub - types.",
+  "allOf": [
+    {
+      "$ref": "_common-wireframe-schema.json"
+    },
+    {
+      "properties": {
+        "shapeStyle": {
+          "type": "object",
+          "description": "The style of this wireframe.",
+          "properties": {
+            "fillColor": {
+              "type": "string",
+              "description": "The filling color for this wireframe as a String hexadecimal. Follows the #RRGGBBAA color format, default value should be #FFFFFF00.",
```

Comment:
just curious why `rgba` over `argb` (the latter is used on Android)?

Reply:
I think the `RRGGBBAA` is more popular